stubs/docutils/docutils/utils/__init__.pyi

```diff
@@ -1,16 +1,17 @@
 import optparse
-from _typeshed import Incomplete
-from builtins import list as _list  # alias to avoid name clashes with fields named list
-from collections.abc import Iterable
-from typing import Literal
+from _typeshed import StrPath
+from collections.abc import Callable, Iterable, Mapping
+from re import Pattern
+from typing import IO, Any, Literal, TypeVar
 from typing_extensions import TypeAlias
 
-from docutils import ApplicationError
-from docutils.io import FileOutput
+from docutils import ApplicationError, DataError, nodes
+from docutils.frontend import Values
+from docutils.io import ErrorOutput, FileOutput
 from docutils.nodes import document
 
 class DependencyList:
-    list: _list[str]
+    list: list[str]
```

Comment:
I personally feel this shadowing shouldn't be allowed but I suppose type checkers allow it.